docs/en_US/reference/experiment_config.rst

```diff
@@ -482,6 +494,10 @@ Hostname of OpenPAI service.
 
 type: ``str``
 
+This may includes ``https://`` or ``http://`` prefix.
+
+HTTPS will be used by default.
```

Comment:
Seems default `http`, [refer](https://github.com/microsoft/nni/blob/master/ts/nni_manager/training_service/pai/paiTrainingService.ts#L50)

Reply:
The config class will add `https://` if no protocol specified.